monkey/monkey_island/cc/resources/island_logs.py

```diff
@@ -1,19 +1,26 @@
 import logging
+from pathlib import Path
 
 from monkey_island.cc.resources.AbstractResource import AbstractResource
 from monkey_island.cc.resources.request_authentication import jwt_required
-from monkey_island.cc.services.island_logs import IslandLogService
 
 logger = logging.getLogger(__name__)
 
 
 class IslandLog(AbstractResource):
-    # API Spec: Why the inconsistency in endpoints of IslandLog and Log?
-    urls = ["/api/log/island/download"]
+    urls = ["/api/island/log"]
+
+    def __init__(self, island_log_file_path: Path):
+        self._island_log_file_path = island_log_file_path
 
     @jwt_required
     def get(self):
         try:
-            return IslandLogService.get_log_file()
+            return self._get_log_file_contents()
         except Exception:
             logger.error("Monkey Island logs failed to download", exc_info=True)
+
+    def _get_log_file_contents(self):
+        with open(self._island_log_file_path, "rt") as f:
+            log_file = f.read()
+        return {"log_file": log_file}
```

Comment:
This is the format of the response. Response formats should always be defined in the root method.